aten/src/ATen/native/Activation.cpp

```diff
@@ -371,4 +371,13 @@ Tensor hardshrink_backward_cpu(const Tensor & grad, const Tensor & self, Scalar
   return out_tensor;
 }
 
+Tensor gelu_cpu(const Tensor& self) {
+  const auto X = self.contiguous();
```

Comment:
why?

Reply:
is this `.contiguous()` still needed after your latest changes?

Reply:
I think so, since currently our approach is using for-loop to take advantage of autovectorization for the non-mkl path. Actually currently with or without MKL, the performance is pretty similar.